dx_app/src/code.sh

```diff
@@ -105,14 +105,6 @@ main() {
   echo "      - Output directory: $OUT_DIR"
   echo ""
 
-
-  echo "  [*] Loading container image ..."
-  image_tarfile_path=/stjude/docker/cicero-docker.tar
-  if [ -e $image_tarfile_path.gz ]
-  then gunzip $image_tarfile_path.gz
-  fi
-  docker load -i $image_tarfile_path
-   
   echo "  [*] Determine image ID ..."
   image_id=`docker images -q stjude/cicero | head -n 1`
```

Comment:
Should this be pointed at docker hub?

Reply:
Thanks, good catch. That was legacy code as we used to package the Docker image in the app and then loaded from the tarball. Now The run command points to the tagged version in GitHub packages.